### ISSUE TITLE
Update Astyle to 3.4.10

### ISSUE THREAD

**Issue by axla-io** [LLNL/libROM#245]
Can we update Astyle to  the latest version?
Then it would be much easier to install on mac:
`brew install astyle`

**Comment by dreamer2368** [comment]
Our docker image is based on ubuntu, where the most up-to-date version from `apt-get` is 3.1.

**Comment by axla-io** [comment]
Is astyle available in the docker image?

**Comment by dreamer2368** [comment]
Yes. You can use either `ghcr.io/llnl/librom_env:latest` for x86_64, or `ghcr.io/llnl/librom_env:arm64` for apple chip architecture.

**Comment by axla-io** [comment]
That's great! I wasn't aware
